examples/set-webhook.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+include __DIR__ . '/basics.php';
+
+use React\EventLoop\Factory;
+use unreal4u\TelegramAPI\Abstracts\TelegramTypes;
+use unreal4u\TelegramAPI\HttpClientRequestHandler;
+use unreal4u\TelegramAPI\Telegram\Methods\SetWebhook;
+use unreal4u\TelegramAPI\TgLog;
+
+$loop = Factory::create();
+$tgLog = new TgLog(BOT_TOKEN, new HttpClientRequestHandler($loop));
+
+$webhook = new SetWebhook();
+$webhook->url = 'https://example.com/';
+$webhook->max_connections = 3;
+$webhook->allowed_updates = ['message'];
+$webhook->drop_pending_updates = true;
+$webhook->secret_token = 'some-very-secret-string';
+
+$promise = $tgLog->performApiRequest($webhook);
+$promise->then(
+    function (TelegramTypes $response) {
+        var_dump('Webhook was successfully set. Run get-webhookinfo.php to get detailed information about webhook from server.');
```

Comment:
This could be turned into an echo instead

Reply:
Converted to echo, @NanoSector.